src/utils/CheckBox.js

```diff
@@ -20,15 +20,20 @@ class CheckBox extends Component {
 
   getIcon() {
     if (this.props.checked)
-      return (<Icon name={'md-checkmark'} color={'#44acff'} size={20} />);
+      return (<Icon name={'md-checkmark'} color={'#44acff'}
+        size={(this.props.iconSize) ? this.props.iconSize : 20} />);
     else
       return;
   }
 
   render() {
     let backColor = this.props.checked  ? '#fafafa' : '#ffffff';
-
-    const { iconStyle, labelStyle } = this.props;
+    const {
```

Comment:
actually this style is from react native dropdown module.
if I change this style, there are many jobs.
Do you think this work is needed?